examples/glue_benchmark/glue.py

```diff
@@ -23,86 +27,39 @@
 
 import keras_nlp
 
-FLAGS = flags.FLAGS
+seed = 42
+os.environ["PYTHONHASHSEED"] = str(seed)
+tf.random.set_seed(seed)
+
 
 flags.DEFINE_string(
-    "task_name",
+    "task",
     "mrpc",
     "The name of the GLUE task to finetune on.",
 )
-
-flags.DEFINE_integer(
-    "batch_size",
-    32,
-    "Batch size of data.",
-)
-
-flags.DEFINE_integer(
-    "epochs",
-    2,
-    "Number of epochs to run finetuning.",
-)
-
-flags.DEFINE_float(
-    "learning_rate",
-    5e-5,
-    "Learning rate",
-)
-
+flags.DEFINE_string("model", None, "The Model you want to train and evaluate.")
 flags.DEFINE_string(
-    "tpu_name",
+    "preset",
     None,
-    "The name of TPU to connect to. If None, no TPU will be used. If you only "
-    "have one TPU, use `local`",
+    "The model preset(eg. For bert it is 'bert_base_en', 'bert_tiny_en_uncased')",
 )
```

Comment:
We need another flag `mixed_precision_policy `, see: [link](https://github.com/keras-team/keras-nlp/blob/23e0508feb5002c02e9c91dce2742feb1460ae6b/keras_nlp/benchmarks/sentiment_analysis.py#L37)